sureal/routine.py

```diff
@@ -71,6 +71,9 @@ def _get_reconstruction_stats(raw_scores, rec_scores):
     raw_score_cmap = kwargs['raw_score_cmap'] if 'raw_score_cmap' in kwargs else cmap_factory('red2green')
 
     raw_score_residue_range = kwargs['raw_score_residue_range'] if 'raw_score_residue_range' in kwargs else [None, None]
+
+    sort_quality_scores = kwargs['sort_quality_scores'] if 'sort_quality_scores' in kwargs else False
+
```

Comment:
add an assertion of the expected type `assert isinstance(sort_quality_scores, bool)` to avoid surprises.